.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "percona-xtradb-cluster-galera"]
 	path = percona-xtradb-cluster-galera
-	url = https://github.com/percona/galera
-	branch = 4.x-8.0
+	url = https://github.com/dutow/galera
```

Comment:
https://github.com/percona/galera

Reply:
I will update this after the galera commit is merged. I needed this change for jenkins testing